src/components/Shared/Tables/FilterableTableComponent.tsx

```diff
@@ -353,7 +353,7 @@ export class FilterableTableComponent extends React.Component<FilterableTableCom
         return (
             <Table2
                 className={className}
-                ref={table.updateTableRef ? ref => table.updateTableRef(ref) : null}
+                ref={table.updateTableRef ?? null}
```

Comment:
From [React documentation](https://react.dev/reference/react-dom/components/common#ref-callback):

> React will also call your ref callback whenever you pass a different ref callback. In the above example, (node) => { ... } is a different function on every render. When your component re-renders, the previous function will be called with null as the argument, and the next function will be called with the DOM node.

Therefore, `FilterableTableComponent` updates the ref whenever the table re-renders because there is an additional arrow function in the ref callback.

Moreover, the ref is set to null and then to the DOM node in the process. Before react 18, we use the ref after the two calls are complete. With react 18, the re-render timing somehow changes and we use the ref between the two calls —— the ref is (temporarily) null and the auto scroll is not triggered.

I remove the additional arrow function to avoid unnecessary reset of the ref, avoiding the entire issue.